src/plots/cartesian/graph_interact.js

```diff
@@ -129,6 +144,11 @@ fx.init = function(gd) {
             maindrag.onmouseout = function(evt) {
                 if(gd._dragging) return;
 
+                // When the mouse leaves this maindrag, unset the hovered subplot.
+                // This may cause problems if it leaves the subplot directly *onto*
+                // another subplot, but that's a tiny corner case at the moment.
+                gd._fullLayout._hoversubplot = null;
```

Comment:
Inset plots seem like a not-so-tiny corner case. But does this actually cause problems? Presumably `mouseout` happens before `mouseover` in that case?